dataflow/gpu-workers/README.md

```diff
@@ -0,0 +1,168 @@
+# Workers with GPUs
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=dataflow/gpu-workers/README.md)
+
+This tutorial shows you how to use GPUs on Cloud Dataflow to process
+[Landsat satellite images](https://cloud.google.com/storage/docs/public-datasets/landsat)
+and render them as JPEG files.
+
+## Before you begin
+
+Follow the
+[Getting started with Cloud Dataflow](../README.md)
+page, and make sure you have a Google Cloud project with billing enabled
+and a *service account JSON key* set up in your `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+Additionally, for this sample you need the following:
+
+1. [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=cloudbuild.googleapis.com)
+    Cloud Build.
+
+1. Create a Cloud Storage bucket.
+
+   ```sh
+   export BUCKET=your-gcs-bucket
+   gsutil mb gs://$BUCKET
+   ```
+
+1. Clone the `python-docs-samples` repository.
+
+    ```sh
+    git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+    ```
+
+1. Navigate to the sample code directory.
+
+   ```sh
+   cd python-docs-samples/dataflow/gpu-workers
+   ```
+
+1. Create a Python 3.6 virtual environment and activate it.
```

Comment:
Unfortunately the tensorflow image we're using as a base comes with Python 3.6 preinstalled, so we don't have an option on which version to use. The alternative is using the nvidia image directly and installing a specific Python version manually. I tried it with miniconda and it works, but it feels a little hacky.

On the bright side, I asked the team and they are planning to upgrade the images to Ubuntu 20.04 LTS early next year, so it will come with Python 3.8. So we'll have to update this whenever that happens.